code/windows_stub/stubs.cpp

```diff
@@ -77,7 +77,7 @@ SCP_string dump_stacktrace()
 	
 	// iterate over the returned symbol lines. skip the first, it is the
 	// address of this function.
-	for (int i = 1; i < numstrings; i++)
+	for (decltype(numstrings) i = 1; i < numstrings; i++)
```

Comment:
Maybe a comment could be added to explain why we need that `decltype` stuff? Otherwise someone could remove this again in the future.

Reply:
Makes sense.